Makefile

```diff
@@ -137,7 +105,7 @@ ifneq ($(GITHUB_BASE_REF), false)
 	# Check that the following diff will exit with 0 or 1
 	git diff --name-only FETCH_HEAD || test $? -le 1 || exit 1
 	# Make the dependencies if any changed files are requirements files, otherwise exit
-	git diff --name-only FETCH_HEAD | grep '^requirements/' || exit 0 && $(MAKE) deps
+	git diff --name-only FETCH_HEAD | grep '^requirements/' || exit 0 && bin/deps
```

Comment:
So, that part would not be done inside Docker, right ?

Reply:
Correct, and great point. I moved this out of the Makefile entirely, and also caught that I missed running this in CI.